web/public/css/base/_alert.less

```diff
@@ -0,0 +1,61 @@
+.alert {
+  border: none !important;
+  box-shadow: @cd-box-shadow;
+  box-sizing: border-box;
+  .fa {
+    font-size: 1.5em;
```

Comment:
I'd rather just use the fa classes for this kinda thing instead, so we're consistent with sizes